client/src/components/Libraries/LibrariesList.vue

```diff
@@ -158,6 +158,7 @@
                                     id="paginationPerPage"
                                     autocomplete="off"
                                     type="number"
+                                    onkeyup="if(this.value<0)this.value=1"
```

Comment:
I guess 0 is already something we don't want to allow ?
```suggestion
                                    onkeyup="if(this.value<1)this.value=1"
```
Can you also add `min="1"` ? Ideally that would prevent entering negative value < 1, but I think that might be a browser restriction ? At least it prevents going down with the arrow symbols. I haven't seen a bug report bootstrap-vue, maybe it would be worth opening one ?

Reply:
This may be me being paranoid but also notice that you can write the *'number'* `e` in the input. It seems `type="number"` accepts any decimal number, etc. Is it possible to limit it only to whole integers? Maybe change the open textbox to a dropdown with a couple of fixed values like [10, 50, 100]? This last option may also fix another small issue, the fact that when you want to write for example `10`, as soon as you type the first `1` the pagination changes to 1 element and so on... but that can also be avoided by triggering the change only when pressing enter or leaving the textbox.

Reply:
> Is it possible to limit it only to whole integers

I will try